launch/launch/actions/group_action.py

```diff
@@ -54,6 +58,16 @@ def __init__(
         else:
             self.__launch_configurations = {}
 
+    @staticmethod
+    def parse(entity: Entity, parser: Parser):
+        """Return `GroupAction` action and kwargs for constructing it."""
+        _, kwargs = super(GroupAction, GroupAction).parse(entity, parser)
```

Comment:
@ivanpauno I think I've seen enough of this idiom to think it'd be better to use class methods, which then allows:
```python
super(GroupAction, cls).parse(entity, parser)
```
which IMHO is slightly less terse.

Reply:
Yeah, I don't know why I didn't use classmethods.
You can use `super()` without any arguments in them, which looks better.